.devcontainer/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/vscode/devcontainers/python:3.8

```

Comment:
We actually don't need this file. We can do everything we need inside the devcontainer.json